work_queue/src/pbs_submit_workers

```diff
@@ -21,7 +21,7 @@ pbs_parameters=""
 
 parse_arguments()
 {
-	if [ -z "$cores" -o $cores=0 ]
+	if [ -z "$cores" -o $cores = 0 ]
```

Comment:
if [ -z "$cores" -o [ $cores = 0 ]
The added [ allows it to work in both case but throws a shell error [: too many arguments

Reading into a solution for this.